samples/hello-world/index.js

```diff
@@ -0,0 +1,84 @@
+// This file will only work on node v8.x since it uses async/await.
+// A version of this script is available for node v6.x in index.v6.js
+
+/*eslint node/no-unsupported-features: ["error", {version: 8}]*/
```

Comment:
It would be great to add a comment with the high level goal of the file, usage, and a link to the README.

Reply:
Done